server/services/datacommons.py

```diff
@@ -23,23 +23,9 @@
 
 import requests
 from werkzeug.utils import import_string
+import lib.config as libconfig
 
-if os.environ.get('FLASK_ENV') == 'test':
-    cfg = import_string('configmodule.TestConfig')()
-elif os.environ.get('FLASK_ENV') == 'webdriver':
-    cfg = import_string('configmodule.WebdriverConfig')()
-elif os.environ.get('FLASK_ENV') == 'production':
-    cfg = import_string('configmodule.ProductionConfig')()
-elif os.environ.get('FLASK_ENV') == 'staging':
-    cfg = import_string('configmodule.StagingConfig')()
-elif os.environ.get('FLASK_ENV') == 'autopush':
-    cfg = import_string('configmodule.AutopushConfig')()
-elif os.environ.get('FLASK_ENV') == 'minikube':
-    cfg = import_string('configmodule.MinikubeConfig')()
-elif os.environ.get('FLASK_ENV') == 'gke':
-    cfg = import_string('configmodule.GKEConfig')()
-else:
-    cfg = import_string('configmodule.DevelopmentConfig')()
```

Comment:
I'm not sure if this could be a problem, but it looks like libconfig.get_config() will raise a ValueError if no FLASK_ENV specified,  but here, the developmentConfig is used if no FLASK_ENV specified

Reply:
Fallback to developmentConfig was okay at the beginning but as we add more config it is better to make it explicit. This consolidation also helps on it as the server should always have a valid FLASK_ENV.

Reply:
Makes sense, thanks for the explanation!